pywikibot/families/liquipedia_family.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+
+from pywikibot import family
+import json
+import requests
+
+
+class Family(family.Family):
+
+    @classmethod
+    def __post_init__(self):
+        response = requests.get('https://liquipedia.net/api.php?action=listwikis', headers={'accept-encoding': 'gzip'})
```

Comment:
A bit of error handling (otherwise you won't get a readable error when you are e.g. ratelimited upon starting the bot)
```suggestion
        response = requests.get('https://liquipedia.net/api.php?action=listwikis', headers={'accept-encoding': 'gzip'})
        if response.status_code != 200:
                print(response.text)
                return
```